### ISSUE TITLE
Demo License not working?

### ISSUE THREAD

**Issue by xAstroBoy** [Polarmods/DevX-Cracked#24]
I put demo license code, but it says "restart application" and then it asks it again.

**Comment by dj3underscores** [comment]
In my case it says “Not activated!” and if I close this window, the only this left is to restart…
And the it happens again endlessly…

**Comment by OrtonLongGaming2** [comment]
yeah, same happening to me. "restart" and then endless loop of having to put it in, told to restart, restart, have to put it in again

**Comment by giganticlab** [comment]
Same happening to me, "license valid" but not activated.

**Comment by Keyden** [comment]
same happening to me. how to resolve it , please fix

**Comment by zhenyao2008** [comment]
同样的情况发生在我身上。“重新启动”，然后无限循环必须将其放入

**Comment by leonpano2006** [comment]
i can't get demo license due to temporary email addresses
i wonder what is not temporary email addresses

**Comment by rem02** [comment]
> i can't get demo license due to temporary email addresses i wonder what is not temporary email addresses

Same problem, do you have a solution ?
